src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -50,7 +50,7 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
-    messageEntity.setProperty("recipient", message.getRecipient());
+    messageEntity.setProperty("postID", message.getRecipient());
```

Comment:
can we change the Message class as well as other occurrences of "recipient" to "postID" in our codebase?

Reply:
actually lets do that in a follow up PR.